maven/JarAssembler.kt

```diff
@@ -58,6 +58,9 @@ class JarAssembler : Callable<Unit> {
     @Option(names = ["--jars"], split = ";")
     lateinit var jars: Array<File>
 
+    @Option(names = ["--fail_on_duplicate_entry"])
+    var failOnDuplicateEntry: Boolean = true
```

Comment:
Given that the duplicate JAR entry problem has now been solved by ensuring the duplicates don't make it in in the first place, I think we should err on the side of caution and remove this `failOnDuplicateEntry` option for now.

Reply:
Agreed. Will remove from this PR and will re-open if I end up needing this again, but from what it seems like, the use case stems from an edge-case project structure and could be avoided by restructuring the project.